packages/instant/src/util/gas_price_estimator.ts

```diff
@@ -0,0 +1,44 @@
+import { BigNumber } from '@0x/utils';
+
+import { DEFAULT_GAS_PRICE } from '../constants';
+
+interface GasStationResult {
+    average: number;
+    fastestWait: number;
+    fastWait: number;
+    fast: number;
+    safeLowWait: number;
+    blockNum: number;
+    avgWait: number;
+    block_time: number;
+    speed: number;
+    fastest: number;
+    safeLow: number;
+}
+
+const endpointUrl = 'https://ethgasstation.info/json/ethgasAPI.json';
+const fetchFastAmountInWei = async () => {
+    const res = await fetch(endpointUrl);
+    const gasInfo = (await res.json()) as GasStationResult;
+    const gasPriceInGwei = new BigNumber(gasInfo.fast / 10);
```

Comment:
why are we dividing by ten here? let's leave a comment

Reply:
also we might want to be safe and grab `fast` using `_.get` just in case there is a breaking change to the API

Reply:
I'm down to clean this up to be more understandable, but FWIW this was straight up ported from code already in website 😆 https://github.com/0xProject/0x-monorepo/blob/development/packages/website/ts/blockchain.ts#L941

Reply:
If the API changes this should just raise an error and fallback to the `catch` in the function that uses this, so not sure if adding a `_.get` is necessary, I kind of like the "let it fail" approach here.
That being said, in the future might be good to report to rollbar or some sort of metric tracking every time a failure happens here, so we would know if the API changed